hugo/layouts/_default/single.html

```diff
@@ -3,18 +3,17 @@
 
     <article class="article">
         <div class="article__container">
-            <header class="article__header">
-                <h1 class="article__title">{{ .Title }}</h1>
-                {{- with .Params.meta -}}
-                <div class="article__meta">{{ partial "meta" . }}</div>
-                {{- end }}
-            </header>
+            {{- partial "article--header" . -}}
             <div class="article__content">
                 {{- .Content -}}
             </div>
+            <footer class="article__footer">
+                {{- if .Params.tags -}}
+                    {{- partial "tags.html" (dict "modifier" "large" "context" . ) -}} 
```

Comment:
don't pass the full context to tags: partials should not be dependent on whatever context they are in but should be usable anywhere